source/blog/2018-08-03-the-ember-times-issue-58.md

```diff
@@ -22,8 +22,10 @@ Check out what's been going in Emberland 🐹 these days:
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [The Quest for Better Documentation](https://github.com/emberjs/website/pull/3481)
+There’s a [brand new RFC](https://github.com/jenweber/rfcs-1/blob/cli-guides/active/0000-cli-guides.md) for restructuring and refreshing the Ember CLI docs. [Jen Weber](https://twitter.com/jwwweber?lang=da) prepared a great overview of what needs to change to make **Ember CLI documentation great again**. This is especially important since these change could empower new contributors, correct outdated information and make new the learning process way easier.
```

Comment:
I think the link to Jen's Twitter could be replaced with the one that falls back to the user's browser language:

```
https://twitter.com/jwwweber
```

What do you think?